customTests/azure_cpu_stream.nhc

```diff
@@ -3,6 +3,8 @@
 # Run Stream
 
 EXE_DIR=$AZ_NHC_ROOT/bin
+source customTests/azure_common.nhc
```

Comment:
Please use this instead: /etc/nhc/scripts/azure_common.nhc

When we build the docker container this is where the file will/should reside. This is probably a good time to look into setting up a test ACR so we can build and publish test docker images.

Reply:
I have made changes on the different test files using /etc/nhc/scripts/azure_common.nhc now